hibernate-reactive/src/main/resources/META-INF/native-image/io.micronaut.sql/hibernate-jpa-graal/reflect-config.json

```diff
@@ -1,16 +1,116 @@
 [
-  {
-    "name": "org.hibernate.internal.log.ConnectionPoolingLogger_$logger",
-    "condition": {
-      "typeReachable": "org.hibernate.internal.log.ConnectionPoolingLogger"
-    },
-    "methods": [
-      {
-        "name": "<init>",
-        "parameterTypes": [
-          "org.jboss.logging.Logger"
+    {
```

Comment:
When a stable version of hibernate-reactive gets released, I will create metadata for it in the shared metadata repository and then remove this reflect-config.json file.